docs/en/ingest-management/release-notes/release-notes-7.16.asciidoc

```diff
@@ -56,6 +56,31 @@ Fleet::
 * Improves default settings for {fleet} component templates {kib-pull}114101[#114101]
 * Report `installing` status while package install is in progress {kib-pull}111875[#111875]
 
+[discrete]
+[[known-issues-7.16.x]]
+=== Known issues
+
+{kib-pull}121059[#121059]
+Error when creating an Agent Policy, seen after editing an Integration policy that is used in other Agent Policies
+[%collapsible]
+====
+
+*Details*
+
+After editing a System integration policy used by other Agent Policies, if a new Agent Policy is created, error "There is already a package with the same name" is thrown.
```

Comment:
The details here are hard to follow. I can't review/edit this without understanding this better. Can you ping me and go over the details? It sounds like it's related to integration policy names needing to be unique. Is the code not smart enough to give the system integration a unique name when you try to create an agent policy?  I'm not sure what editing an existing system integration policy has to do with this problem. A 5-minute chat would probably clear this up.

Reply:
Hi DeDe, sure! I'll ping you and explain what happens.